src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -0,0 +1,242 @@
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import { FeatureFlag } from '@app/Shared/FeatureFlag/FeatureFlag';
+import { dashboardConfigTemplateHistoryClearIntent } from '@app/Shared/Redux/ReduxStore';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { FeatureLevel } from '@app/Shared/Services/Settings.service';
+import { portalRoot } from '@app/utils/utils';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  Gallery,
+  KebabToggle,
+  Label,
+  Split,
+  SplitItem,
+  Title,
+} from '@patternfly/react-core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { getConfigByName } from './Dashboard';
+import {
+  CardConfig,
+  iconify,
+  LayoutTemplate,
+  LayoutTemplateContext,
+  LayoutTemplateVendor,
+  SerialCardConfig,
+} from './dashboard-utils';
+
+export interface LayoutTemplateGroupProps {
+  title: string;
+  templates: LayoutTemplate[];
+  onTemplateSelect: (templateName: LayoutTemplate) => void;
+  onTemplateDelete: (templateName: string) => void;
+}
+
+export const smallestFeatureLevel = (cards: SerialCardConfig[]): FeatureLevel => {
+  return cards.reduce((minFeatureLevel: FeatureLevel, card: CardConfig) => {
+    const featureLevel = getConfigByName(card.name).featureLevel;
+    return Math.min(minFeatureLevel, featureLevel);
+  }, FeatureLevel.PRODUCTION);
+};
+
+export const LayoutTemplateGroup: React.FC<LayoutTemplateGroupProps> = ({
+  onTemplateSelect,
+  onTemplateDelete,
+  ...props
+}) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { selectedTemplate, setSelectedTemplate } = React.useContext(LayoutTemplateContext);
+  const scrollRef = React.useRef<HTMLDivElement>(null);
+
+  const handleTemplateSelect = React.useCallback(
+    (template: LayoutTemplate) => {
+      setSelectedTemplate(template);
+      onTemplateSelect(template);
+    },
+    [setSelectedTemplate, onTemplateSelect]
+  );
+
+  const handleClearRecent = React.useCallback(() => {
+    dispatch(dashboardConfigTemplateHistoryClearIntent());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Split>
+        <SplitItem>
+          <Title headingLevel="h2" size="lg" style={{ padding: '1em' }}>
+            {props.title} ({props.templates.length})
```

Comment:
How about adding the word `items`?

![Screenshot from 2023-04-12 21-03-14](https://user-images.githubusercontent.com/68053619/231619038-c2649ce5-edc7-4667-8b75-6671b782d832.png)